filters/ratelimit/ratelimit.go

```diff
@@ -230,15 +215,18 @@ func clusterClientRatelimitFilter(args []interface{}) (filters.Filter, error) {
 	}
 
 	if len(args) > 3 {
-		headerName, err := getStringArg(args[3])
+		lookuperString, err := getStringArg(args[3])
 		if err != nil {
 			return nil, err
 		}
-		headerName = http.CanonicalHeaderKey(headerName)
-		if headerName == "X-Forwarded-For" {
-			s.Lookuper = ratelimit.NewXForwardedForLookuper()
+		if strings.Contains(lookuperString, ",") {
```

Comment:
maybe this could be simpler. What if we don't check whether the lookuperString contains a ",", but we just use strings.Split immediately. If there is no "," in it, then it will return a slice of one item. Then we can check if the slice contains more items.

Reply:
As discussed offline:
Contains vs. creating a slice (memory garbage).
I am more for the Contains in this case.